data/types/devicestatus/status/signalstrength.go

```diff
@@ -0,0 +1,35 @@
+package status
+
+import (
+	"github.com/tidepool-org/platform/data"
+	"github.com/tidepool-org/platform/structure"
+)
+
+type SignalStrength struct {
+	Unit  *string  `json:"units,omitempty" bson:"units,omitempty"`
+	Value *float64 `json:"value,omitempty" bson:"value,omitempty"`
+}
+
+func ParseSignalStrength(parser structure.ObjectParser) *SignalStrength {
+	if !parser.Exists() {
+		return nil
+	}
+	datum := NewSignalStrength()
+	parser.Parse(datum)
+	return datum
+}
+
+func NewSignalStrength() *SignalStrength {
+	return &SignalStrength{}
+}
+
+func (s *SignalStrength) Parse(parser structure.ObjectParser) {
+	s.Unit = parser.String("unit")
+	s.Value = parser.Float64("value")
+}
+
+func (s *SignalStrength) Validate(validator structure.Validator) {
+}
```

Comment:
`unit` and `value` should be required - that's my mistake. I only just thought that through, sorry :(

Reply:
Added tests for unit and value